Common/Brokerages/BybitBrokerageModel.cs

```diff
@@ -0,0 +1,198 @@
+﻿/*
+* QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+* Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using QuantConnect.Benchmarks;
+using QuantConnect.Orders;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Securities;
+using QuantConnect.Util;
+
+namespace QuantConnect.Brokerages;
+
+/// <summary>
+/// Provides Bybit specific properties
+/// </summary>
+public class BybitBrokerageModel : DefaultBrokerageModel
+{
+    /// <summary>
+    /// Market name
+    /// </summary>
+    protected virtual string MarketName => Market.Bybit;
+
+    /// <summary>
+    /// Gets a map of the default markets to be used for each security type
+    /// </summary>
+    public override IReadOnlyDictionary<SecurityType, string> DefaultMarkets { get; } = GetDefaultMarkets(Market.Bybit);
+
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="BybitBrokerageModel"/> class
+    /// </summary>
+    /// <param name="accountType">The type of account to be modeled, defaults to <see cref="AccountType.Cash"/></param>
+    public BybitBrokerageModel(AccountType accountType = AccountType.Cash) : base(accountType)
```

Comment:
If margin account type isn't supported atm we should throw an exception if we detect it here 👍